2-mandatory/5-eligible-students.js

```diff
@@ -2,12 +2,27 @@
   Only students who have attended enough classes are eligible to sit an exam.
 
   Create a function which:
-  - Accepts an array which contains all the students' names and their attendance counts
+  - Accepts an array which contains all the students' names and their 
+    attendance counts
     (see tests to confirm how this data will be structured)
-  - Returns an array containing only the names of the who have attended AT LEAST 8 classes
+  - Returns an array containing only the names of the who have attended
+    AT LEAST 8 classes
 */
 
-function getEligibleStudents() {}
+function getEligibleStudents(arrays) {
+    //create an empty array to store the arr who will sit an exam
+     const sitExamName = [];
+     //loop through the array of arrays
+     for ( arr of arrays ){
+    // check if the array includes a name with 8 attendance counts
+       if(arr[1] >= 8){
+        //add the arr to the sitExamName array
+        sitExamName.push(arr)
```

Comment:
You could have already done: `sitExamName.push(arr[0])` and then you would not need to use map in line 24.